src/providers/credential/OpenId4VcIssuanceProvider.ts

```diff
@@ -255,16 +254,20 @@ class OpenId4VcIssuanceProvider {
     if (!this.credentialBranding) {
       this.credentialBranding = new Map<string, Array<IBasicCredentialLocaleBranding>>();
       await Promise.all(
-        this.credentialsSupported.map(async (metadata: ICredentialMetadata): Promise<void> => {
+        this.credentialsSupported.map(async (metadata: CredentialSupported): Promise<void> => {
           const localeBranding: Array<IBasicCredentialLocaleBranding> = await Promise.all(
             (metadata.display ?? []).map(
-              async (display: CredentialDisplay): Promise<IBasicCredentialLocaleBranding> =>
+              async (display: MetadataDisplay): Promise<IBasicCredentialLocaleBranding> =>
                 await ibCredentialLocaleBrandingFrom({localeBranding: await credentialLocaleBrandingFrom(display)}),
             ),
           );
 
+          const credentialTypes: Array<string> = metadata.types.length > 1
+            ? metadata.types.filter((type: string) => type !== 'VerifiableCredential')
+            : metadata.types
```

Comment:
Probably wise to also check for an empty array, as then the result should be an array with only VerifiableCredential in there

Reply:
added a fix for that